packages/gestalt-design-tokens/README.md

```diff
@@ -0,0 +1,181 @@
+# Basic Style Dictionary
```

Comment:
@ayeshakmaz could we update this file and make it specific to Gestalt? Perhaps we can use a link to the `style-dictionary` docs?

Reply:
Totally, will do 👍